server/protocols/jmap-rfc-8621-integration-tests/memory-jmap-rfc-8621-integration-tests/pom.xml

```diff
@@ -42,12 +42,6 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>${james.groupId}</groupId>
-            <artifactId>james-server-jmap-rfc-8621</artifactId>
-            <type>test-jar</type>
-            <scope>test</scope>
-        </dependency>
```

Comment:
This dependency introduced a wrong version of javax.activation on the class path

Reply:
can we exclude the dependency, that makes it incompatible?

Reply:
I did not find it, I just would like to fix the build quickly.

Reply:
Feel free to look for it...